qoqo/src/operations/mod.rs

```diff
@@ -148,5 +150,9 @@ pub fn operations(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add_class::<EchoCrossResonanceWrapper>()?;
     m.add_class::<PragmaAnnotatedOpWrapper>()?;
     // 1.9
```

Comment:
Could you check if anything was added in 1.9?

Reply:
Both from the Changelog and from the `ImplementedIn1Point9` trait it looks like nothing was added